wordpress-mobile/WordPress-Android
# [Revamp Landing Screen] Fix system bars background transparency

Fixes #17284 on both WordPress and Jetpack apps.

The implementation uses the new `Window.setDecorFitsSystemWindows(boolean)` API for Android 11+ and falls back to the initial setup of using `LayoutParams.FLAG_LAYOUT_NO_LIMITS` for older Android version.

I've tested on my Android 9 device and noticed the issue wasn't reproducing by following the repro steps.

To test:
1. Open the app
2. Press the Enter your existing site address
3. Press back
5. Lock the screen
6. Unlock the screen
7. **Expect** no white bars at the top and bottom of the app

## Regression Notes
1. Potential unintended areas of impact
   None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
   None

3. What automated tests I added (or what prevented me from doing so)
   None

PR submission checklist:
